content/en-us/reference/engine/classes/SoundService.yaml

```diff
@@ -86,8 +86,29 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: SoundService.DefaultListenerLocation
-    summary: ''
-    description: ''
+    summary: |
+      Determines where (if anywhere) to place an `Class.AudioListener` by default.
+    description: |
+      Determines where to place an `Class.AudioListener` by default.
+      If `Class.SoundService.DefaultListenerLocation` is `None`, then no `Class.AudioListener` will be created by default, but `Class.AudioListeners` can be created separately by scripts.
+      If `Class.SoundService.DefaultListenerLocation` is `Camera`, then
+      - an `Class.AudioListener` will be created and parented-to `Class.Workspace.CurrentCamera`
+      - an `Class.AudioDeviceOutput` will be created and parented-to `Class.SoundService`
+      - a `Class.Wire` will be created and parented-to the previously-created `Class.AudioListener`
+      - `Class.Wire.SourceInstance` will be set to the previously-created `Class.AudioListener`, and `Class.Wire.TargetInstance` will be set to the previously-created `Class.AudioDeviceOutput`
```

Comment:
The documentation supports nested lists, right? We could indent this one forward to put it under the previous line about the wire

Reply:
Hm I would think you indent the bullet one more time but I don't know how legit that is